submissions/pozzitive11/js-world-oop/index.js

```diff
@@ -0,0 +1,65 @@
+class Inhabitant {
+  constructor(species, name, gender, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+  }
+
+  getProperties() {
+    return `${this.species}; My name ${this.name}; ${this.gender}; ${this.saying}`;
+  }
+}
+class Animal extends Inhabitant {
+  constructor(name, gender, saying) {
+    super("animal", name, gender, saying);
+    this.paw = 4;
+  }
+  getProperties() {
+    return super.getProperties() + `; I have ${this.paw} paws;`;
+  }
+}
+
+class Human extends Inhabitant {
+  constructor(name, gender, saying) {
+    super("human", name, gender, saying);
+    this.hands = 2;
+    this.legs = 2;
+  }
+
+  getProperties() {
+    return (
+      super.getProperties() +
+      `; I have ${this.legs} legs and ${this.hands} hands;`
+    );
```

Comment:
Why body of return statement is enclosed with parentheses?

Reply:
VSCode has interesting auto-formatting

Reply:
Not critical, but check if you have enforced JSX ruleset applied to all projects by default.